exercises/book-store/src/test/java/BookStoreTest.java

```diff
@@ -0,0 +1,126 @@
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Test;
+
+public class BookStoreTest{
+
+	@Test
+	public void Only_a_single_book(){
+		Integer[] p = {1};
+		ArrayList<Integer> books = new ArrayList<>(Arrays.asList(p));
```

Comment:
Normally when we declare an object like this in Java, we prefer to declare using the [interface](http://stackoverflow.com/a/383982/3084820).  In this case:

```Java
List<Integer> books = new ArrayList<>(Arrays.asList(p));
```